ReSwift.xcodeproj/xcshareddata/xcbaselines/25DBCF861C30C4DB00D63A58.xcbaseline/Info.plist

```diff
@@ -28,6 +28,30 @@
 			<key>targetArchitecture</key>
 			<string>x86_64</string>
 		</dict>
+		<key>CA98BC4B-4A03-4B07-BF9E-12C3F6A52195</key>
+		<dict>
+			<key>localComputer</key>
+			<dict>
+				<key>busSpeedInMHz</key>
+				<integer>100</integer>
+				<key>cpuCount</key>
+				<integer>1</integer>
+				<key>cpuKind</key>
+				<string>Intel Core i7</string>
+				<key>cpuSpeedInMHz</key>
+				<integer>2800</integer>
+				<key>logicalCPUCoresPerPackage</key>
+				<integer>8</integer>
+				<key>modelCode</key>
+				<string>MacBookPro14,3</string>
+				<key>physicalCPUCoresPerPackage</key>
+				<integer>4</integer>
+				<key>platformIdentifier</key>
+				<string>com.apple.platform.macosx</string>
+			</dict>
+			<key>targetArchitecture</key>
+			<string>x86_64</string>
+		</dict>
 	</dict>
```

Comment:
Seems like performance test artifacts slipped in by mistake?

Reply:
The performance test artifacts are necessary for testing changes to the codebase, as the baselines are needed to see if there are differences in performance. Anyone with the same computer as the baselines set will get a test failure if they regress performance.
I like these being committed as a historical record of where the test was performed & what the result was.